src/utils/Processing.ts

```diff
@@ -47,21 +47,32 @@ export function findInputDir(input: string): string {
   return input;
 }
 
-export function ensureOutputDir(input: string, output: string, isIgPubContext: boolean): string {
-  if (isIgPubContext) {
+export function ensureOutputDir(
+  input: string,
+  output: string,
+  isIgPubContext: boolean,
+  isLegacyIgPubContext: boolean
+): string {
+  if (isIgPubContext || isLegacyIgPubContext) {
     // TODO: Message includes information about legacy support for top level fsh folder. Remove when not supported.
+    const directory = isIgPubContext ? 'input/fsh' : 'fsh';
     logger.info(
-      'SUSHI detected an "input/fsh" or "fsh" directory in the input path. As a result, SUSHI will operate in "IG Publisher integration" mode. This means:\n' +
-        '  - the "input/fsh" or "fsh" directory will be used as the input path\n' +
-        '  - the parent of the "fsh" directory (e.g., "../fsh") will be used as the output path unless otherwise specified with --out option\n' +
+      `SUSHI detected an "${directory}" directory in the input path. As a result, SUSHI will operate in "IG Publisher integration" mode. This means:\n` +
```

Comment:
If the directory is `fsh`, will this end up saying SUSHI detected `an fsh`?

Reply:
uh oh. It will. I'll update it.